src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -2027,21 +2462,21 @@ internal static bool SkipSystemRestoreOperationForARMPlatform(PSCmdlet cmdlet)
         }
 
         /// <summary>
-        /// Invokes the Win32Shutdown command on provided target computer using WSMan
+        /// Invokes the Win32ShutdownTracker command on provided target computer using WSMan
         /// over a CIMSession.  The flags parameter determines the type of shutdown operation
         /// such as shutdown, reboot, force etc.
         /// </summary>
         /// <param name="cmdlet">Cmdlet host for reporting errors</param>
         /// <param name="isLocalhost">True if local host computer</param>
         /// <param name="computerName">Target computer</param>
-        /// <param name="flags">Win32Shutdown flags</param>
+        /// <param name="flags">Win32ShutdownTracker flags</param>
         /// <param name="credential">Optional credential</param>
         /// <param name="authentication">Optional authentication</param>
         /// <param name="formatErrorMessage">Error message format string that takes two parameters</param>
         /// <param name="ErrorFQEID">Fully qualified error Id</param>
         /// <param name="cancelToken">Cancel token</param>
         /// <returns>True on success</returns>
-        internal static bool InvokeWin32ShutdownUsingWsman(
+        internal static bool InvokeWin32ShutdownTrackerUsingWsman(
```

Comment:
I don't think it's a good idea to change the method name itself. If a new overload or a new method that does what you need is necessary, that's probably better, unless you're absolutely sure that nothing else calls this method at any point. It's flagged as `internal`, so potentially anything in the same namespace could call it, though I'm not sure if anything currently does.

Reply:
There's no specific reason why it needs to be changed, it's just to match the name of the new CimOperatingSystemShutdownMethod. I can revert to the previous name as I don't think an overload is necessary.

Reply:
Gotcha. Yeah, if you can check the references to that method to see if anything else happened to also use it that would be best.

Otherwise I'll be more than happy to do a quick check in a bit just to make sure. 😆

Reply:
Just did a quick check, and it doesn't look like anything else is using this method, so it's probably fine either way. 😄

Reply:
Oh great, thanks. I might leave it as is then to match the new shutdown method name 😀